### ISSUE TITLE
[0.9.3] inserting NaNs causes value to become unqueryable.

### ISSUE THREAD

**Issue by DanielMorsing** [influxdata/influxdb#4104]
```
> create database foo
> use foo
Using database foo
> insert cpu value=200
> insert cpu foobar=nan
> select * from cpu
ERR: invalid character 'j' looking for beginning of value
> select value from cpu
name: cpu
---------
time				value
2015-09-15T15:18:10.512161841Z	200

```

**Comment by timraymond** [comment]
@DanielMorsing To help with reproducing this issue, could you prefix the title of this issue with the version of Influx that you're encountering this under (e.g. `[0.x.x] inserting NaNs...`)?

Also helpful information would be the OS that you're using and whether this is a cluster or single node setup.

**Comment by malaakso** [comment]
I have the same problem: 0.9.3, Linux, single node.

**Comment by beckettsean** [comment]
Similar to https://github.com/influxdb/influxdb/issues/3994

**Comment by luciotorre** [comment]
Fwiw, I have the same problem: 0.9.4, Linux, single node.

**Comment by rajeshmurali** [comment]
select * from cpu where value>=0 works,
Case 2:
Now try to insert a value for foobar field
example: insert cpu foobar=123
this query is successful
now try to fetch the values from foobar field by using the query 
select * from cpu where foobar>=0

using the query select * from cpu is confusing influxdb or influxdb could not display a field with Not a Number (NaN) value

**Comment by jwilder** [comment]
Fixed via 0926b19
